### ISSUE TITLE
Make sure XSS is impossible

### ISSUE THREAD

**Issue by staab** [cesardeazevedo/nostr-editor#4]
Since we're going to be using nostr-editor to render notes from other people, we should be sure XSS isn't possible. I'm unclear from [the docs](https://tiptap.dev/docs/guides/output-json-html) whether this is the case. See also https://github.com/ueberdosis/tiptap/pull/5160

**Comment by cesardeazevedo** [comment]
Indeed, I also got rid of our own link extension in favor of extending tiptap link-extension because of that, and apparently tiptap already handles it https://github.com/chroth/tiptap/blob/main/tests/cypress/integration/extensions/link.spec.ts#L74

Also tiptap-markdown uses markdown-it under the hood https://github.com/markdown-it/markdown-it/security

Planning to add some more unit/e2e tests soon

**Comment by staab** [comment]
I'm no longer planning to use the editor to render stuff, so feel free to close this.

**Comment by cesardeazevedo** [comment]
Me neither, I use yours welshman/content for a while now
